libs/utils/automated-aria.js

```diff
@@ -153,8 +153,7 @@ const tryAssignAriaLabel = (
   });
 
   const buttonText = cta.textContent.trim().toLowerCase();
-  const productInCTA = cta.classList.contains('modal') ? ''
-    : getProduct(cta.href.replace('-', ' '), productNames);
+  const productInCTA = !cta.classList.contains('modal') && !!cta.href ? getProduct(cta.href.replace('-', ' '), productNames) : '';
```

Comment:
The `!!` is not needed and imho makes it harder to read.

Another option would be to not use the ternary - but I think removing !! is enough:
```
  const productInCTA = (
    !cta.classList.contains('modal')
    && cta.href
    && getProduct(cta.href.replace('-', ' '), productNames)
  ) || '';
```

Reply:
it's a good point, I'll do that :)